orc/src/main/java/org/apache/iceberg/orc/ORCSchemaUtil.java

```diff
@@ -383,7 +383,7 @@ private static boolean isSameType(TypeDescription orcType, Type icebergType) {
         .map(Integer::parseInt);
   }
 
-  static int fieldId(TypeDescription orcType) {
+  public static int fieldId(TypeDescription orcType) {
```

Comment:
Can we avoid making this public with the other updates?

Reply:
I think `GenericOrcWriter` is still currently using it and it's in `org.apache.iceberg.data.orc` but this method is in `org.apache.iceberg.orc`, so we will still have to declare public...